### DISCUSSION TITLE
Temporarily Rescinding Account Redirect to Download Data?

### DISCUSSION THREAD

**Discussion by rickscully** [mastodon/mastodon#18163] [General]
I decided to sunset my one-person instance and move to a new community. A few days ago I set the old instance to redirect to my new instance. I have since learned about downloading the data, but it looks as if I would need to cancel the re-direct to get access to that feature. 

**Question:** What are the ramifications -- if any -- of turning off redirecting to get the data and then redirecting again after the data has been downloaded?

Thanks.

**Comment by Gargron** [comment]
No ramifications. The redirect is basically just a banner, it is not the same thing as moving followers (though moving followers automatically enables the redirect).

**Reply by rickscully** [reply]
Thanks. My concern was whether the system would attempt to move followers a second time.

**Reply by rickscully** [reply]
Follow up. I see I had already downloaded all of my followers, blocks, bookmarks etc.  Is there a way to download my posts before I cancel my hosting and blow the whole instance away?

**Reply by rickscully** [reply]
never mind. That's what is in the tar I already downloaded. Thanks again!
